dyc/base.py

```diff
@@ -33,7 +33,7 @@ def initialize(self, change=None):
                         if word.lstrip() in keywords
                     ]
                 )
-                > 0
+                > 0 and line.lstrip(' ')[0] != '#' # line is not a comment line
```

Comment:
> 0 and line.lstrip(' ').startswith("#")

Reply:
Hey guys, I think this attempt prevents the ability of identifying comments by other languages.

Preferable to have

in `config/defaults.yaml`
Add a `comments` list that takes as a start `['"""', '#', "'''"]`
Then update that here

Reply:
Yes. That would be a general solution.